src/lib/prisma.ts

```
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client';
+
+const globalForPrisma = globalThis as unknown as {
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)